Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.7
+
+WORKDIR /app
+
+RUN apk add --no-cache wget ca-certificates && \
+    wget -O /etc/apk/keys/phpearth.rsa.pub https://repos.php.earth/alpine/phpearth.rsa.pub && \
+    echo "https://repos.php.earth/alpine/v3.7" >> /etc/apk/repositories && \
+    apk add --no-cache --update certbot nginx dcron curl tini php7.2 php7.2-bcmath php7.2-common php7.2-dom php7.2-fpm php7.2-gd php7.2-mbstring php7.2-openssl php7.2-zip php7.2-pdo php7.2-phar php7.2-json php7.2-pdo_mysql php7.2-session php7.2-ctype php7.2-tokenizer php7.2-zlib php7.2-simplexml && \
```

Comment:
Alpine Linux 3.8 got PHP 7.2 packages (worked on it myself).

If you want the latest PHP 7.2 version then I suggest that this uses the official PHP image as the base - https://hub.docker.com/_/php/